src/OpenTracing/Carriers/TextMap.php

```diff
@@ -17,7 +17,11 @@ private function __construct(array $textMap)
         }
     }
 
-    public static function create(array $textMap = [])
+    /**
+     * @param array|string[] $textMap
+     * @return TextMap
+     */
+    public static function fromMap(array $textMap = [])
```

Comment:
I find the term "Map" here confusing, I would think of an SPL `Map`, not an associative array

Reply:
Makes totally sense. I just changed it.